src/index.js

```diff
@@ -1,127 +1,92 @@
+        const addNumberElement = document.querySelector("p")
+        addNumberElement.innerText = "Add Number"
 
-    // MAIN
+        // MAIN
+        const mainElement = document.createElement("main")
+        const buttonWrapperElement = document.createElement("div")
+        buttonWrapperElement.id = "button-wrapper"
 
-    const mainElement = document.createElement("main")
-    const buttonWrapperElement = document.createElement("div")
-    buttonWrapperElement.id = "button-wrapper"
-    const firstLineWrapper = document.createElement("div")
-    firstLineWrapper.className = "line-wrapper"
-    const button1Element = document.createElement("button")
-    button1Element.id = "button-1"
-    button1Element.innerText = "1"
-    const button2Element = document.createElement("button")
-    button2Element.id = "button-2"
-    button2Element.innerText = "2"
-    const button3Element = document.createElement("button")
-    button3Element.id = "button-3"
-    button3Element.innerText = "3"
-    const secondLineWrapper = document.createElement("div")
-    secondLineWrapper.className = "line-wrapper"
-    const button4Element = document.createElement("button")
-    button4Element.id = "button-4"
-    button4Element.innerText = "4"
-    const button5Element = document.createElement("button")
-    button5Element.id = "button-5"
-    button5Element.innerText = "5"
-    const button6Element = document.createElement("button")
-    button6Element.id = "button-6"
-    button6Element.innerText = "6"
-    const thirdLineWrapper = document.createElement("div")
-    thirdLineWrapper.className = "line-wrapper"
-    const button7Element = document.createElement("button")
-    button7Element.id = "button-7"
-    button7Element.innerText = "7"
-    const button8Element = document.createElement("button")
-    button8Element.id = "button-8"
-    button8Element.innerText = "8"
-    const button9Element = document.createElement("button")
-    button9Element.id = "button-9"
-    button9Element.innerText = "9"
-    const fourthLineWrapper = document.createElement("div")
-    fourthLineWrapper.className = "line-wrapper"
-    const buttonStarElement = document.createElement("button")
-    buttonStarElement.id = "star"
-    buttonStarElement.innerText = "*"
-    const button0Element = document.createElement("button")
-    button0Element.id = "button-0"
-    button0Element.innerText = "0"
-    const buttonHashElement = document.createElement("button")
-    buttonHashElement.id = "hash"
-    buttonHashElement.innerText = "#"
-    const fifthLineWrapper = document.createElement("div")
-    fifthLineWrapper.className = "line-wrapper"
-    const buttonCallElement = document.createElement("button")
-    buttonCallElement.id = "call"
-    buttonCallElement.innerText = "Call"
-    const buttonXElement = document.createElement("button")
-    buttonXElement.id = "button-x"
-    buttonXElement.innerText = "X"
+        headerElement.appendChild(mainElement)
```

Comment:
Main should be on the same level as header, so you cannot append main as a header child